research/src/sources/web.json5

```diff
@@ -93,11 +93,18 @@
     },
     {
       "name": "Img",
-      "url": "https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element"
-    },
-    {
-      "name": "Img",
-      "url": "https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element"
+      "openUIName": "Image",
+      "url": "https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element",
+      "concepts": [
+        {
+          "name": "width",
```

Comment:
The width and height image examples are of API docs. It would be ideal to have component specimens instead. I can see why this would be difficult to capture as there isn't really a great visual for width/height.

If there aren't actual visual specimens for width/height I wonder if we should exclude them? It just seems strange to see API docs in the visual examples area.

![image](https://user-images.githubusercontent.com/5067638/110860438-9e14a300-8271-11eb-8bff-ccd767ad3cae.png)

Reply:
Yep, great points, thanks @levithomason! I was on the fence whether to include them myself, so happy to remove them since they don't really add much value!